linode/manager
# Add GooglePay logic, make a one time payment

## Description

- Adds Braintree SDK
- Adds Braintree logic to handle Google Pay payments
- Adds new hook `useScript` and `useLazyScript`
- Adds a Google Pay Button for one-time payments (will go in the `Make a Payment` drawer)
- Adds Google Pay button for adding Google pay as a payment method (commented out for now, will be handled in a follow up PR by @bnussman )
- All Google Pay related changes are behind a feature flag
- Fix existing formatting bug in make a payment drawer where clicking on the arrows in the amount input changes the formatting from 2 decimal to none. (e.g. 5.00 -> 5)

## How to test
Use dev services, In Account -> Make a payment click on the GPay button. The google pay flow modal should open, go through the flow and click Pay (you won't be charged in dev). There should be a success notification at the top of the drawer.
